docs/why-stacked.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_label: "Why Stacked"
+sidebar_position: 0
+---
+
+# What is Stacked
+
+Stacked is the production framework for Flutter. For teams or solo developers that require production level code, Stacked makes it easy to achieve. [FilledStacks](https://www.youtube.com/filledstacks) is a mobile software development agency with experience using Native, Xamarin and now Flutter to develop over 30 applications for large companies. We know what is required to build big, **scalable**, **testable** and **maintanable applications**, this is the core focus of Stacked.
+
+- **Scalable**: Stacked is built to make your team scalable and keep your productivity high. With strict principles and a strong opinion on how to develop functionality, you or your team will have a clear guide around adding or maintaining features.
+
+- **Testable**: We put an emphasis on unit tests and our architecture is desiged to make unit testing any part of your business logic or state as easy as possible.
```

Comment:
here MVCC, why its good and how we adhere to it

Reply:
You've mentioned MVCC above as well. Stacked is based on MVVM. Just want to get insight into why you're calling it MVCC. I'll update to MVVM for now since that's been the communication from the start. The final part to make it full MVVM is two-way binding which we'll add in as well to remove some boilerplate code.

Reply:
I meant MVVM :see_no_evil: